src/amo/components/AddonReview/index.js

```diff
@@ -164,30 +164,41 @@ export class AddonReviewBase extends React.Component<Props, State> {
     }
 
     let placeholder;
-    let prompt;
+    let promptText;
     if (review.rating && review.rating > 3) {
-      prompt = i18n.gettext(
-        'Tell the world why you think this extension is fantastic!'
+      promptText = i18n.gettext(
+        'Tell the world why you think this extension is fantastic! Please read our %(linkStart)sreview guidelines%(linkEnd)s first.'
```

Comment:
This is more than 80chars :)

Reply:
I forgot that we use babel to extract gettext messages so it's probably safe to use back ticks. I'll update it.